robotiq_hande_driver/include/protocol_logic.hpp

```diff
@@ -115,7 +112,11 @@ class ProtocolLogic{
      * @return none
      * @note see status on success, exception thrown if communicatoin issues
      */
-    void set();
+    void set() {
+        communication_.clear_output_bytes();
+        communication_.write_action_bit((uint)ActionRequestPositionBit::ACTIVATE, (bool)Activate::ACTIVATE_GRIPPER);
```

Comment:
Consider to change the underlying type of the `enum class`es  `ActionRequestPositionBit` and `Activate` items' to `uint` and `bool` respectively.
```suggestion
        communication_.write_action_bit(static_cast<uint>(ActionRequestPositionBit::ACTIVATE), static_cast<bool>(Activate::ACTIVATE_GRIPPER));
```